requirements.txt

```diff
@@ -17,6 +19,10 @@ lxml==4.9.1
     #   dbus-deviation
     #   feedgen
     #   hotdoc (setup.py)
+networkx==2.5
```

Comment:
These are are versions that were accidentally removed in previous PR. Let's update them in follow-up instead.

Reply:
oof, I should have paid more attention, thanks :)